seqr/fixtures/1kg_project.json

```diff
@@ -295,7 +295,7 @@
         "last_modified_date": "2017-03-12T19:27:08.773Z",
         "project": 1,
         "family_id": "10",
-        "display_name": "10",
+        "display_name": "10_a",
```

Comment:
Is there any reason to change this one?

Reply:
So right now, the results are sorted strictly by the length of the title, so when the query is "1" then  "2_1" is sorted before "10_a" because of length. In the coming ticket I am going to change the behavior to prioritize first character match wherever possible, so "10_a" would come before "2_1" so I needed to add fixture data so that change would be testable. I updated the tests to explicitly check the order all the results are returned in so that this will be clearer

Reply:
also, I realized I actually needed a different field for this so I changed which family has the long display name, its family 12 now